test/JDBC/expected/BABEL_1940.out

```diff
@@ -0,0 +1,224 @@
+
+-- Test is only valid when default server encoding is WIN1252
+SELECT CONVERT(VARCHAR(10), 0x123456789)
+GO
+~~START~~
+varchar
+#Eg‰
+~~END~~
+
+
+SELECT CONVERT(VARBINARY(10), '#Eg‰')
+GO
+~~START~~
+varbinary
+0123456789
+~~END~~
+
+
+SELECT CONVERT(VARCHAR(1), 0x99)
+GO
+~~START~~
+varchar
+™
+~~END~~
+
+
+SELECT CONVERT(VARCHAR(2), 0x999999)
+GO
+~~START~~
+varchar
+™™
+~~END~~
+
+
+SELECT CONVERT(VARBINARY(1), '™')
+GO
+~~START~~
+varbinary
+99
+~~END~~
+
+
+SELECT CAST('™' AS VARBINARY)
+GO
+~~START~~
+varbinary
+99
+~~END~~
+
+
+SELECT CAST('™™™' AS VARBINARY(2))
+GO
+~~START~~
+varbinary
+9999
+~~END~~
+
+
+SELECT CONVERT(VARCHAR(10), 0x80)
+GO
+~~START~~
+varchar
+€
+~~END~~
+
+
+-- 0x81 does not exist is empty in some encodings
+SELECT CONVERT(VARCHAR(10), 0x81)
+GO
+~~START~~
+varchar
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: character with byte sequence 0x81 in encoding "WIN1252" has no equivalent in encoding "UTF8")~~
```

Comment:
This error is probably not right behaviour, we should either replace it to '?' or '0x81' depending on where this error occurring

Reply:
adding a missing flag to conversion functions ?